src/clarity-angular/dropdown/dropdown-submenu.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, ContentChildren, Input, QueryList} from "@angular/core";
+import {DropdownItem} from "./dropdown-item";
+
+@Component({
+    selector: "clr-dropdown-submenu",
+    template: `
+        <ng-content></ng-content>
+    `,
+    host: {
+        "[class.dropdown-submenu]" : "true",
+        "[class.open]" : "open",
+        "[class.right]" : "openDirection === 'right'",
+        "[class.left]" : "openDirection === 'left'",
+        "(click)": "open = true",
+        "(mouseenter)": "open = true",
+        "(mouseleave)": "open = false"
+    }
+})
+export class DropdownSubmenu {
+    private _open: boolean = false;
+    @Input ("clrDropdownSubmenuOpenDirection") openDirection: string = "right";
+
+    @ContentChildren(DropdownItem, {descendants: false}) menuItems: QueryList<DropdownItem>;
```

Comment:
Wow, descendants is now true by default? It was false not long ago, that's actually a breaking change on their part. When did it happen, 4.0?